docs/index/secondary-index-guide.md

```diff
@@ -183,6 +183,9 @@ Syntax
   ```
   DROP INDEX [IF EXISTS] index_name ON [TABLE] [db_name.]table_name
   ```
+Note: If a column to be dropped has any Secondary index table created on them, drop column 
```

Comment:
please add this note in drop column section

Reply:
added